apps/fabric-website/src/SiteDefinition/SiteDefinition.pages/Styles/web.tsx

```diff
@@ -141,9 +141,9 @@ export const stylesPagesWeb: INavPage[] = [
       require.ensure([], require => cb(require<any>('../../../pages/Styles/LocalizationPage/LocalizationPage').LocalizationPage))
   },
   {
-    title: 'Fabric 7 Updates',
+    title: 'Fluent UI 7 Updates',
```

Comment:
I wonder if this one should still be Fabric? (since 6 is definitely Fabric, and this page is relevant for people upgrading from 6)

Reply:
in this case it's about the changes in "Fluent UI 7" which seems ok to me. I guess when talking about 5x and 6x we would refer to that as Fabric?

Reply:
Yeah thinking about it again that should be okay. 5 and 6 should definitely still be referred to as Fabric.